modules/sdk/gradle-plugins-workspace/build.gradle

```diff
@@ -119,6 +118,7 @@ dependencies {
 
 	compileInclude files("lib/com.liferay.workspace.bundle.url.codec-1.0.0.jar")
 
+	compileOnly group: 'com.google.code.gson', name: 'gson', version: '2.9.0'
```

Comment:
@jeffwu0724 You changed gradle plugin related modules, please check with Peter Shin if you need to include the fake gradle cache.